src/IOST/Account.cpp

```diff
@@ -4,18 +4,26 @@
 using namespace TW;
 using namespace TW::IOST;
 
-bool Account::isValid(const std::string& s) {
-    if (s.size() < 5 || s.size() > 11) { 
-        return false; 
+bool isAccountValid(const std::string& account) {
+    if (account.size() < 5 || account.size() > 11) {
+        return false;
     }
-    for (char ch : s) {
+    for (char ch : account) {
         if ((ch < 'a' || ch > 'z') && (ch < '0' || ch > '9') && ch != '_') {
             return false;
         }
     }
     return true;
 }
 
+bool Account::isValid(const std::string& s) {
+    if (s.size() == 44) {
```

Comment:
just remove this check

Reply:
remove this check will cause CoinAddressDerivationTests() to fail for IOST, as address validation will fail to derived address (e.g. "EKRQPgX7nKt8hJABwm9m3BKWGj7kcSECkJnCBauHQWin")

Reply:
let's check if it's a valid base58 string then